prow/plugins/reward-owners/reward-owners.go

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rewardowners
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+
+	"k8s.io/test-infra/prow/plugins/ownersconfig"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	verifyowners "k8s.io/test-infra/prow/plugins/verify-owners"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/git/v2"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const (
+	// PluginName defines this plugin's registered name.
+	PluginName    = "reward-owners"
+	RewardMessage = `
+Thanks for serving the community in this new capacity!
+
+Next steps:
+1- Reach out in #sig-contribex for your special badge swag!
```

Comment:
Can we add a link to the channel?

Reply:
I think that would be:
```suggestion
1- Reach out in [#sig-contribex](https://app.slack.com/client/T09NY5SBT/C1TU9EB9S) for your special badge swag!
```

... Except the user also needs to be a member of the slack, so we might want to instead mention  https://slack.k8s.io in this comment or some doc about joining the slack if there is one.